src/screens/IndexScreen.js

```diff
@@ -248,6 +250,7 @@ export const IndexScreen = ({ navigation, route }) => {
                     <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
                   ) : null
                 }
+                ListEmptyComponent={listItems ? <EmptyMessage title={texts.empty.list} /> : null}
```

Comment:
istn't `ListEmptyComponent` implicitly checking the data for presence and therefore we could get rid of `listItems ?...` and pass our `EmptyMessage` directly?

Reply:
At the moment, I tried it as you said. I did not encounter any error. This is how I am sending an update.

Reply:
Should I use it with `<SafeAreaViewFlex>` ?

Reply:
as the list is rendered already inside the wrappers, i think we do not need anything here.